testgrid/config/config.yaml

```diff
@@ -2674,78 +2581,35 @@ dashboards:
   - name: periodic-bazel-test-1.7
     test_group_name: periodic-kubernetes-bazel-test-1-7
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
 - name: release-1.7-blocking
   dashboard_tab:
-  - name: build-1.7
-    test_group_name: ci-kubernetes-build-1.7
+  - name: gce-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-release-1-7
+  - name: gce-serial-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-serial-release-1-7
+  - name: gce-slow-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-slow-release-1-7
+  - name: gce-reboot-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-reboot-release-1-7
+  - name: gce-scalability-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-scalability-release-1-7
   - name: test-go-1.7
     test_group_name: ci-kubernetes-test-go-release-1.7
   - name: verify-1.7
     test_group_name: ci-kubernetes-verify-release-1.7
   - name: kubelet-1.7
     test_group_name: ci-kubernetes-node-kubelet-1.7
-  - name: federation-build-1.7
-    test_group_name: ci-kubernetes-federation-build-1.7
-  - name: gce-release-1-7
-    test_group_name: ci-kubernetes-e2e-gce-release-1-7
-  - name: gci-gce-release-1-7
```

Comment:
Sure?  This is up to all of you @kubernetes/kubernetes-release-managers.  Remember the goal here when coming up with a set of HARD gating tests, is that you can get a passing set with some regularity as the point is to block the release.  Don't add anything here that's just "been failing forever".  And the suites may differ between branches (master vs. release-*).